modules/simple_forms_api/app/models/simple_forms_api/vba_21_0966.rb

```diff
@@ -93,6 +99,82 @@ def track_user_identity(confirmation_number)
 
     private
 
+    def veteran_ssn
+      [
+        data.dig('veteran_id', 'ssn')&.[](0..2),
+        data.dig('veteran_id', 'ssn')&.[](3..4),
+        data.dig('veteran_id', 'ssn')&.[](5..8)
+      ]
+    end
+
+    def veteran_date_of_birth
+      [
+        data['veteran_date_of_birth']&.[](0..3),
+        data['veteran_date_of_birth']&.[](5..6),
+        data['veteran_date_of_birth']&.[](8..9)
+      ]
+    end
+
+    def veteran_address
+      [
+        data.dig('veteran_mailing_address', 'postal_code')&.[](0..4),
+        data.dig('veteran_mailing_address', 'postal_code')&.[](5..8)
+      ]
+    end
+
+    def veteran_home_phone
+      [
+        data['veteran_phone']&.gsub('-', '')&.[](0..2),
+        data['veteran_phone']&.gsub('-', '')&.[](3..5),
+        data['veteran_phone']&.gsub('-', '')&.[](6..9)
```

Comment:
do we also want to include 'veteran_international_phone'?

Reply:
Same reasoning as va file number, it gets plopped in wholesale so we don't need to specifically call it out here.